index.js

```diff
@@ -7,7 +7,7 @@ const childProcess = require("child_process");
 const version = process.argv[2] || "latest";
 
 const packageJsonPath = path.join(process.cwd(), "package.json");
-if (!fs.existsSync(path.join(process.cwd(), "package-lock.json"))) {
+if (!fs.existsSync(packageJsonPath)) {
```

Comment:
doh!  Good catch - that's what I get for literally never using `yarn` in my own apps